src/controllers/__tests__/project.test.ts

```diff
@@ -16,13 +16,27 @@ describe("get project", () => {
   });
 
   test("Passed appropriate query - it should successfully fetch existing project", async () => {
+    // Since mockProject doesn't align to response of the endpoint
+    // This variable is to set the same shape as the response body
+    const expectedReturn = {
+      id: mockProject.id,
+      name: mockProject.name,
+      description: mockProject.description,
+      columns: [
+        { ...mockProject.columns[0], tickets: [] },
+        { ...mockProject.columns[1], tickets: [] },
+      ],
+    };
```

Comment:
This is a bit patchy, but I needed to make a shape of `expectedReturn` since `mockProject` isn't in the shape of the response of the endpoint.